src/psen_scan_udp_interface.cpp

```diff
@@ -55,37 +55,29 @@ void PSENscanUDPInterface::write(const boost::asio::mutable_buffers_1& buffer)
   socket_write_.send(buffer);
 }
 
+bool PSENscanUDPInterface::isUdpMsgAvailable() const
+{
+  return socket_read_.available() > 0u;
+}
+
 /**
  * @brief Receive data from the scanner.
  *
  * @param buffer Boost receive buffer class.
  * @return std::size_t Returns how many bytes have been read.
- *
- * @throws UDPReadTimeoutException
```

Comment:
The function seems to me like it can still throw this exception

Reply:
The function is trivial. Therefore, I see no reason to have a information duplication represented by this documentation in the first place. Furthermore, this documentation should be included in the header and, more over, in the interface class and not in the derived class. I, therefore, decided to move the documentation to the interface class.

Reply:
Done.